docs/pages/server-access/guides/openssh.mdx

```diff
@@ -340,6 +340,71 @@ authenticate the host via the certificate we generated earlier.
 
 </Details>
 
+<Details title="Using Proxy Jump in Trusted Clusters?">
```

Comment:
```suggestion
<Details title="Using Trusted Clusters?">
```
Suggested because `tsh config` generates the config with a `ProxyCommand` directive, so this question would always be false for users who have been following the instructions. I think this Details box would be relevant for anyone using Trusted Clusters.